python/lsst/rucio/register/script.py

```diff
@@ -106,6 +106,14 @@ def _register_zips(ri, zip_files, chunk_size, rucio_dataset):
         logger.debug(f"{cnt} zips registered")
 
 
+def _register_dims(ri, dim_files, chunk_size, rucio_dataset):
+    # register dataset_refs with Rucio into the rucio dataset, in chunks
+    for dim_file in dim_files:
+        rp = ResourcePath(dim_file)
+        cnt = ri.register_dims(rucio_dataset, [rp])
+        logger.debug(f"{cnt} dimension files registered")
```

Comment:
No f-strings in log messages please. :wink:

Reply:
fixed